pom.xml

```diff
@@ -596,14 +601,15 @@
         <apache.felix.scr.ds.annotations.version>1.2.4</apache.felix.scr.ds.annotations.version>
         <equinox.osgi.services.version>3.5.100.v20160504-1419</equinox.osgi.services.version>
         <commons-collections.wso2.osgi.version.range>[3.2.0,4.0.0)</commons-collections.wso2.osgi.version.range>
+        <identity.data.publisher.authentication.version>5.1.9-SNAPSHOT</identity.data.publisher.authentication.version>
```

Comment:
Usually we don't rely on snapshot unless it's really required. Can we get this repo released and then update the released version there ? (Only if possible)